src/odemis/acq/stream/_sync.py

```
@@ -565,6 +574,23 @@ def _getScanStagePositions(self):
 
+    def guessFoV(self):
+        """
+        Estimate the field-of-view based on the current settings.
+        :return: (float, float): width, height in meters
+        :raises: ValueError if it cannot be guessed (ie, no stream has guessFoV())
+        """
+        for s in self._streams:
+            try:
+                fov = s.guessFoV()
+                logging.debug("Guessed FoV from stream %s: %s", s, fov)
+                return fov
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)